app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
Thanks for explanation!
As you said, if `SessionModel` is same object, all views receive `onChanged` event.
But now `SessionModel` of `List<SessionModel>` in SessionsFragment and `SessionModel` in SessionDetailFragment are different object.

It might be better to create repository class and cache on memory.

一応日本語でも書いておきます 🙇 
SessionModelのオブジェクトがSessionsViewModelとSessionDetailViewModelで同じなら説明のやり方でモデルの変更をView(Fragment)に通知されるようにできるんですが、Bundleに詰めてIntent経由でオブジェクトを別のActivityに渡すと別オブジェクトになってしまうんですよね。

これを解決するには、Repositoryみたいなシングルトンクラスを作るなどして同じオブジェクトを扱えるようにしておくしかないってことですかね